### PR TITLE
Complete review

### PR DESCRIPTION
**Pull request by VincentBlondeau**

### CODE REVIEW

**Comment by jecisc** [comment]
Thanks. I'll review tomorrow

File: devdocumentation.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h1 class="logo">
     <div class="doc-wrapper">
         <div class="container">
             <div id="doc-header" class="doc-header text-center">
-                <h1 class="doc-title"><i class="icon fa fa-code"></i> Developer Documentation</h1>
+                <h1 class="doc-title"><i class="icon fa fa-code"></i>Developer Documentation</h1>
```
Comment (jecisc) [inline devdocumentation.html]: The space was wanted for design :)
I'll re-add it.

Comment (VincentBlondeau) [reply devdocumentation.html]: That is strange but why not :)

Comment (jecisc) [reply devdocumentation.html]: This is to have a little space between the icon and the title.

**Pull request merged**